version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -167,6 +187,14 @@ public Set<Stability> getStabilitySet() {
         return this.stabilities;
     }
 
+    /**
+     * The product ASCII banner defined in MANIFEST.MF using \n as line ending.
+     * @return the ASCII banner.
+     */
```

Comment:
This is not accurate since the banner is being retrieved from a file and not from the MANIFEST.MF in this class